entrypoint.sh

```diff
@@ -77,15 +78,15 @@ echo "Container Initialization complete."
 echo "Starting Riven (Backend)..."
 if [ "$PUID" = "0" ]; then
     if [ "${DEBUG}" != "" ]; then  # check if DEBUG is set to a truthy value
-        cd /riven/src && poetry add debugpy && poetry run python3 -m debugpy --listen 0.0.0.0:5678 main.py
+        cd /riven/src && poetry add debugpy && poetry run python3 -m debugpy --listen 0.0.0.0:5678 main.py --port $PORT
```

Comment:
We dont need to change the container running port as it doesnt effect the host port anyways. So this isnt needed.

Reply:
this is required if there are multiple instance of riven are running in same shared container network, like in vpn scenario, where instead of relying on proxy, spin up a wireguard container and have all traffice route throuth that container. In that case, port clash is a real issue, kinda setup i'm using.

Reply:
Ahh okay, in that case nvm :D